tests/wpt/tests/tools/wptrunner/wptrunner/executors/executorservo.py

```diff
@@ -27,14 +26,22 @@
 webdriver = None
 
 
-class ServoExecutor(ProcessTestExecutor):
+# A mixin class that includes functionality common to all Servo
+# executors that work by spawing a new process. This is intended to
+# be used along with either the `TestExecutor` class or its children
+# and must be the first in the inheritance list to allow `super`
+# to forward the calls to correct base class.
+class ServoExecutorMixin:
     def __init__(self, logger, browser, server_config, headless,
-                timeout_multiplier, debug_info,
+                 timeout_multiplier, debug_info,
                  pause_after_test, reftest_screenshot="unexpected"):
-        ProcessTestExecutor.__init__(self, logger, browser, server_config,
-                                     timeout_multiplier=timeout_multiplier,
-                                     debug_info=debug_info,
-                                     reftest_screenshot=reftest_screenshot)
+        super().__init__(logger, browser, server_config,
+                         timeout_multiplier=timeout_multiplier,
+                         debug_info=debug_info,
+                         reftest_screenshot=reftest_screenshot)
```

Comment:
Do we still need to call `super().__init__` here since this only extends `Object`?